src/main/kotlin/uk/gov/dwp/dataworks/services/ActiveUserTasks.kt

```diff
@@ -0,0 +1,52 @@
+package uk.gov.dwp.dataworks.services
+
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue
+import uk.gov.dwp.dataworks.UserHasNoTasksException
+import uk.gov.dwp.dataworks.UserTask
+import uk.gov.dwp.dataworks.aws.AwsCommunicator
+import uk.gov.dwp.dataworks.logging.DataworksLogger
+import javax.annotation.PostConstruct
+
+@Component
+class ActiveUserTasks {
+    companion object {
+        val logger: DataworksLogger = DataworksLogger(LoggerFactory.getLogger(ActiveUserTasks::class.java))
+    }
+
+    @Autowired
+    private lateinit var awsCommunicator: AwsCommunicator
+
+    private val dynamoTableName = "UserTasks"
```

Comment:
Not sure if we have a naming policy for DynamoDB, put it may be prudent to prepend the system name to the table to try and implement a pseudo namespace, as UserTasks seems very generic, and doing so would be easy to tie back the table to this service.

Reply:
Good shout, I will rename the table. Also just noticed that I've typo'd the primary key so I'll fix that.
In future it might be a better idea to create the table with TF and pass it in as an env var, seems a little cleaner to me and the less that needs managing by the OS the better imo